### PR TITLE
doc: Add a note for dotfiles on html_static_path (refs: #4859)

### PR DESCRIPTION
**Pull request by tk0miya**
### Feature or Bugfix
- Document

### Purpose
- refs: #4859

### CODE REVIEW

File: doc/config.rst [hunk 1]
```diff
@@ -877,6 +877,18 @@ that use Sphinx's HTMLWriter class.
    named :file:`default.css` will overwrite the theme's
    :file:`default.css`.
 
+   .. note::
+
+      For security reason, dotfiles under ``html_static_path`` will
```
Comment (cpitclaudel) [inline doc/config.rst:882]: typo: `reason` → `reasons`

File: doc/config.rst [hunk 2]
```diff
@@ -877,6 +877,18 @@ that use Sphinx's HTMLWriter class.
...
+      not be copied.  If you'd like to copy them intentionally, please
```
Comment (cpitclaudel) [inline doc/config.rst:883]: typo: `like` → `would like`

File: doc/config.rst [hunk 3]
```diff
@@ -877,6 +877,18 @@ that use Sphinx's HTMLWriter class.
...
+      add them each filepath to this setting::
```
Comment (cpitclaudel) [inline doc/config.rst:884]: typo: `them each` → `each`

File: doc/config.rst (hunk 1 above)
Comment (tk0miya) [reply doc/config.rst:882]: Thanks. Your comment is very helpful to me :-)

Comment (cpitclaudel) [reply doc/config.rst:882]: Thanks for all your work on Sphinx!

**Pull request merged**